src/scripts/main.js

```diff
@@ -354,7 +354,25 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the HTML. If it doesn't, `table` will be `null`, and attempting to append children to it will cause an error. Consider adding a check to ensure `table` is not `null` before proceeding.